packages/app-data/assets/index.ts

```diff
@@ -3170,8 +3170,8 @@ export const ASSETS_CONTENTS_LIST = {
     md5Checksum: "681910eca8201998e670a0f3caffa9e7",
     themeVariations: {
       theme_professional: {
-        size_kb: 12.7,
-        md5Checksum: "340dc0cd22ef431e6d09f9af4b643640",
+        size_kb: 27.9,
```

Comment:
@jfmcquade I added one image and the professional theme asset size more than doubled - does that make sense to you?

Reply:
This is just the size of that single image, namely the professional variation of `plh_images/workshops/w_self_care/guide_2/reward.svg` – each one has a separate entry in this `index.ts` file.

 And looking at the changed image, the new one is significantly more complicated so it being double the size sounds about right:
<img width="500" alt="Screenshot 2022-11-08 at 10 28 48" src="https://user-images.githubusercontent.com/64838927/200541526-9fdb5646-aab9-480d-bf8a-060c63efeb5f.png">

~30kb is very small anyway, one of the advantages of using svg files. A suitably scaled png or jpg might be more like 500kb.

So yes, all good.